testing/dart/compositing_test.dart

```diff
@@ -274,7 +274,13 @@ void main() {
       return builder.pushOpacity(100, oldLayer: oldLayer);
     });
     testNoSharing((SceneBuilder builder, EngineLayer oldLayer) {
-      return builder.pushColorFilter(const Color.fromARGB(0, 0, 0, 0), BlendMode.color, oldLayer: oldLayer);
+      return builder.pushColorFilter(
```

Comment:
Let's add test coverage for all 4 types of `ColorFilter`s. I think they don't have to be put in the `SceneBuilder does not share a layer between addRetained and push*` test. Maybe just add a new test `pushColorFilter handles all 4 types`?

Reply:
Will do